docs/google-kubernetes-tutorial.md

```diff
@@ -101,7 +96,7 @@ We can watch the operator come up with:
 
 	watch kubectl get pods --namespace tidb-admin -o wide
 
-When you see `Running`, `Control + C` and proceed to launch a TiDB cluster!
+When you see all pods are `Running`, `Control + C` and proceed to launch a TiDB cluster!
```

Comment:
Shall we specify whats pods are there? Currently the tidb-scheduler and tidb-controller-manager, otherwise it's not clear what all pods mean.

Reply:
makes sense, I'll update it

Reply:
updated